### ISSUE TITLE
Lack of color contrast in the license selected in `Help` > `Licenses`

### ISSUE THREAD

**Issue by joaopalmeiro** [jupyterlab/jupyterlab#16542]
Hi! 👋 

## Description

In the table on the Licenses page in `Help` > `Licenses`, the font color of the selected package lacks color contrast with the background:

<img width="1582" alt="License table" src="https://github.com/jupyterlab/jupyterlab/assets/17132927/718b58af-12d7-47ed-9fbd-77380cab4a5d">

<img width="1582" alt="Hovered license table" src="https://github.com/jupyterlab/jupyterlab/assets/17132927/50a8a2a2-7109-4193-930b-fcb6ee904563">

## Reproduce

1. Go to `Help` > `Licenses` in the top menu.
2. Select a package from the license table.

## Expected behavior

The font color contrasts with the background, and the background color is blue, similar to what happens in the file browser:

<img width="1582" alt="image" src="https://github.com/jupyterlab/jupyterlab/assets/17132927/c6e7e4cc-937e-4497-bcfd-df6b2bc1f3fc">

Additionally, the background color should not change when hovering over the selected row, similar to what happens in the file browser.

### Possible solutions

To achieve this behavior, I think the [table styles](https://github.com/jupyterlab/jupyterlab/blob/841b339771d15a5ae0f0f63455eb7ed66b56d921/packages/rendermime/style/base.css#L460) (e.g., `.jp-RenderedHTMLCommon tbody tr:nth-child(odd)`) should be applied taking into account whether the `.jp-mod-selected` class is present or not (in the `<tr>` element), similar to what happens in the file browser: https://github.com/jupyterlab/jupyterlab/blob/841b339771d15a5ae0f0f63455eb7ed66b56d921/packages/filebrowser/style/base.css#L304

An alternative is to update the license table styles to make them more _specific_: https://github.com/jupyterlab/jupyterlab/blob/841b339771d15a5ae0f0f63455eb7ed66b56d921/packages/help-extension/style/base.css#L221

### Current workaround for a custom theme

```css
.jp-RenderedHTMLCommon tbody tr.jp-mod-selected:hover,
.jp-RenderedHTMLCommon tbody tr.jp-mod-selected:nth-child(even),
.jp-RenderedHTMLCommon tbody tr.jp-mod-selected:nth-child(odd) {
  background: var(--jp-brand-color1);
}
```

## Context

- Operating System and version: macOS Sonoma 14.5
- Browser and version: Chrome 126.0.6478.127
- JupyterLab version: 4.2.3
- Python version: 3.10.13

Let me know what you think and if I can open a PR. Thanks!

**Comment by JasonWeill** [comment]
@joaopalmeiro Thank you for your contribution! Please open a PR if you have a fix for this issue.

**Comment by joaopalmeiro** [comment]
Thanks! I will work on it.
